tests/test_export.py

```diff
@@ -245,6 +245,20 @@ def test_dict_export(snippetcompiler):
     print(_version, json_value, status, model)
 
 
+def test_export_null_in_dict(snippetcompiler):
+    snippetcompiler.setup_for_snippet(
+        """
+import exp
+
+a = exp::Test2(mydict={"a": null}, mylist=["a","b"])
+"""
+    )
+    _version, json_value, status, model = snippetcompiler.do_export(include_status=True)
+
+    assert len(json_value) == 1
```

Comment:
This doesn't really test the null conversion, does it? Perhaps add a null value to `test_2121_wrapped_proxy_serialize` as well? And potentially remove the print at line 342, looks like I forgot to clean that up when I opened the PR for that test.

Reply:
it tests that is doesn't choke on it.

Reply:
improved this